doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -97,15 +117,20 @@ the following changes are required:
       #--------------------------------------
       suseInsertService sshd
 
-   Also make sure to setup **UseDNS=no** in :file:`etc/ssh/sshd_config`
-   This can be done by an overlay file or clever patching of
-   the file in the above mentioned :file:`config.sh` file.
+   Also make sure to setup **UseDNS=no** in :file:`/etc/ssh/sshd_config`
+   This can be done by an overlay file or by patching the file in the above
+   mentioned :file:`config.sh` file.
 
-6. Configure sudo for Vagrant user
+7. Configure sudo for the vagrant user
+
+   Vagrant expects to have passwordless root permissions via ``sudo`` to be
+   able to setup your box. Please add the following line to
+   :file:`/etc/sudoers` to ensure this:
+
+   .. code::
+
+      vagrant ALL=(ALL) NOPASSWD: ALL
```

Comment:
Hmn, shouldn't the line be added with the command `visudo` instead of adding it with an ordinary editor?

Reply:
Can you do that non-interactively? This whole enumeration describes steps that one has to add to `config.sh` so that Vagrant will be able to boot the box. (If that is not clear from the text, please let me know!)

Reply:
> Can you do that non-interactively?

I'm not sure if that is possible.  Maybe `visudo` is not really needed, but `visudo` advertise itself as "visudo - safely edit the sudoers file". So I thought, it *might* be something useful.

> (If that is not clear from the text, please let me know!)

It was clear to me, so no worries. :wink: 

----

By the way, just as a sidenote: would it make sense to create a file under `/etc/suders.d/` instead of manipulating the file `/etc/sudoers` directly?
One benefit could be, that all the configuration is in *one* file. If you don't need it anymore, you can remove it quite easily.